setup.py

```diff
@@ -22,9 +22,7 @@
               , 'Topic :: Internet :: WWW/HTTP :: WSGI :: Application'
                ]
 
-setup( author = 'Gratipay, LLC'
-     , author_email = 'support@gratipay.com'
```

Comment:
`python2 setup.py sdist bdist_wheel` works fine, the metadata files just have `UNKNOWN` for the author field.